src/main/resources/jenkins/plugins/git/GitPublisherStep/config.jelly

```diff
@@ -0,0 +1,128 @@
+        <f:repeatable field="tagsToPush"
+                      add="${%Add Tag}">
+            <table width="100%">
+                <br/>
+                <f:entry field="tagName"
+                         title="${%Tag to push}">
+                    <f:textbox/>
+                </f:entry>
+                <f:entry field="tagMessage"
+                         title="${%Tag message}">
+                    <f:textarea/>
+                </f:entry>
+                <f:entry field="createTag"
+                         title="${%Create new tag}">
+                    <f:checkbox />
+                </f:entry>
+                <f:entry field="updateTag"
+                         title="${%Update new tag}">
+                    <f:checkbox />
+                </f:entry>
+                <f:entry field="targetRepoName"
+                         title="${%Target remote name}">
+                    <f:textbox/>
+                </f:entry>
+            </table>
+            <div align="right">
+                <input type="button" value="Delete Tag" class="repeatable-delete" style="margin-left: 1em;" />
+            </div>
+        </f:repeatable>
+    </f:entry>
+    <f:entry field="branchesToPush"
+             title="${%Branches}"
+             description="${%Branches to push to remote repositories}">
+        <f:repeatable field="branchesToPush"
+                      add="${%Add Branch}">
+            <table width="100%">
+                <br/>
+                <f:entry field="branchName"
+                         title="${%Branch to push}">
+                    <f:textbox/>
+                </f:entry>
+                <f:entry field="targetRepoName"
+                         title="${%Target remote name}">
+                    <f:textbox/>
+                </f:entry>
+            </table>
+            <div align="right">
+                <input type="button" value="Delete Branch" class="repeatable-delete" style="margin-left: 1em;" />
+            </div>
+        </f:repeatable>
+    </f:entry>
+
+    <f:entry field="notesToPush" title="${%Notes}" description="${%Notes to push to remote repositories}">
+
+        <f:repeatable field="notesToPush" add="${%Add Note}">
+
+            <table width="100%">
+                <br/>
+                <f:entry title="${%Note to push}" field="noteMsg" >
+                    <f:textarea/>
+                </f:entry>
+
+                <f:entry field="targetRepoName" title="${%Target remote name}">
+                    <f:textbox/>
+                </f:entry>
+
+                <f:entry title="${%Note's namespace}" field="noteNamespace">
+                    <f:textbox/>
+                </f:entry>
+
+                <f:entry title="${%Abort if note exists}" field="noteReplace" >
+                    <f:checkbox />
+                </f:entry>
+
+            </table>
+            <div align="right">
+                <input type="button" value="Delete Note" class="repeatable-delete" style="margin-left: 1em;" />
```

Comment:
Hi @iwarapter . There is a Wiki page for it: https://wiki.jenkins-ci.org/display/JENKINS/Internationalization